src/solution/client/src/services/filter-list-service.js

```diff
@@ -0,0 +1,22 @@
+import { FILTER_OPTIONS } from "../app/constants/filter-options";
+
+export function filterByStatus(statusFilter, items) {
+  debugger;
+  switch (parseInt(statusFilter)) {
+    case FILTER_OPTIONS[0].value: {
```

Comment:
better add comment regarding what is `FILTER_OPTIONS[0] / [1] / [2]` or work with a map
```
const FILTER_TYPES = {
ALL: 'all',
DONE: 'done',
UNDONE: 'undone'
} 
```